src/bundle/Resources/config/routing.yml

```diff
@@ -564,7 +564,7 @@ ezplatform.content.preview:
         locationId: ~
 
 ezplatform.content.translate:
-    path: /content/{contentId}/translate/{toLanguageCode}/{fromLanguageCode}
+    path: /content/{contentId}/{locationId}/translate/{toLanguageCode}/{fromLanguageCode}
```

Comment:
Hmm, I don't remember if we gave BC promise on route parameters. If so it's a BC break, if not, then it's fine
POV ping @ezsystems/php-dev-team

Reply:
I am sure we do (or at least did in the past ;) ), as we also did not rename controllers method parameters because of that. Thats why it should have default value and same behavior as current with it.

Reply:
@alongosz, @ViniTou in this case we won't be able to fix this one as we need to `locationId` in order for the proper location to be loaded when translating. Right now this is a decision if we want to keep the BC or fixing this.

Reply:
> @alongosz, @ViniTou in this case we won't be able to fix this one as we need the `locationId` in order for the proper location to be loaded when translating. Right now this is a decision if we want to keep the BC or fix this.

@barw4 Usually I'd encourage you to find on your own a solution which provides both, but since this is CSR, let me just serve it. 
There are 2 ways of providing BC for the route itself:

A) New route:
  1. Create different route
  2. Use different route where needed
  3. If the old route is not needed everywhere, deprecate (if no route deprecation is available, simple place a deprecating comment)

B) Default parameter
  1. Place `{locationId}` at the end instead
  2. Provide default value via `defaults` route attribute

Question: I don't see consumption changes, only passing the parameter. Where is it used?

Reply:
The default parameter option was not possible as we had the other default parameter at the end.